### PR TITLE
Document how the TOC election process works

### PR DESCRIPTION
**Pull request by delvh**
This is supposed to prevent a power vacuum so that a problem similar to the 2024 election does not happen again

Additionally, update current TOC members from 2023 to 2024.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -486,36 +487,53 @@ if possible provide GPG signed commits.
...
+If an elected member that accepts the seat does not have 2FA configured yet, they will be temporarily counted as `answer pending` until they manage to configure 2FA, thus leaving their seat empty for this duration.
 
 ### Current TOC members
 
-- 2023-01-01 ~ 2023-12-31 - https://blog.gitea.com/quarterly-23q1/
+- 2024-01-01 ~ 2024-12-31
   - Company
     - [Jason Song](https://gitea.com/wolfogre) <i@wolfogre.com>
     - [Lunny Xiao](https://gitea.com/lunny) <xiaolunwen@gmail.com>
     - [Matti Ranta](https://gitea.com/techknowlogick) <techknowlogick@gitea.io>
```
Comment (techknowlogick) [inline CONTRIBUTING.md]: Can you change this to use Gitea.com instead of the .io domain?

Comment (delvh) [reply CONTRIBUTING.md]: Sure.
There are a few instances of the `.io` domain, should I fix all of them as well?
On a related note, does `security@gitea.com` exist?

Comment (delvh) [reply CONTRIBUTING.md]: Okay, `try.gitea.io` should perhaps not be renamed.

Comment (techknowlogick) [reply CONTRIBUTING.md]: Oh, there are still many uses of .io that still are in use, I was just meaning in my email.

Comment (delvh) [reply CONTRIBUTING.md]: Done

**Review by khmarbaise — APPROVED** [review]

**Review by kdumontnu — APPROVED** [review]
:+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -486,36 +487,53 @@ if possible provide GPG signed commits.
 https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/
 https://help.github.com/articles/signing-commits-with-gpg/
 
+Furthermore, any account with write access (like bots and TOC members) **must** use 2FA.
```
Comment (denyskon) [inline CONTRIBUTING.md:490]: I would maybe even go as far as require 2FA for every maintainer. After all, it is not that hard with GitHub nowadays....

**Review by denyskon — APPROVED** [review]

**Review by justusbunsi — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (techknowlogick) [reply CONTRIBUTING.md:490]: There is currently a blocker to require this for accessibility reasons and it is presently being investigated.

Comment (denyskon) [reply CONTRIBUTING.md:490]: Oh I was completely sure that GitHub offers email 2FA which should be somehow doable for everyone, but seems like they don't, only SMS messages 😕
Then I have to agree with you

**Review by denyskon — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
PS: my vote goes only to the docs change

for the update that I will be in elected as silverwind replacement, I'm thankfull and do accept it.
but I'd like to see other maintainers lgtm as legitimation not mine ;)

**Review by jolheiser — APPROVED** [review]

**Review by BLumia — APPROVED** [review]

**Review by yp05327 — APPROVED** [review]

**Comment by delvh** [comment]
8 approvals.
I don't think there will be any more.
I'm giving a deadline of 12 hours for further feedback and will then merge the PR if there's no new feedback.

**Pull request merged**